src/models/links/AbstractLinks.php

```diff
@@ -33,10 +33,17 @@ abstract class AbstractLinks extends AbstractRest
 {
     use SetIdTrait;
 
+    /**
+     * The id of the target (link_id)
+     */
+    public ?int $id;
```

Comment:
why not add this in the constructor args?

Reply:
Because `SetIdTrait` is used. The only reason I added the class property is to leverage the docblock. The comment in the constructor was good, but it is even better when the IDE tells us everywhere in the class/codebase that this id is somewhat special.